packages/ozone/src/daemon/context.ts

```diff
@@ -88,6 +91,16 @@ export class DaemonContext {
       cfg.db.materializedViewRefreshIntervalMs,
     )
 
+    const verificationService = VerificationService.creator()
+    const verificationListener = cfg.verifier?.jetstreamUrl
+      ? new VerificationListener(
+          verificationService(db),
+          new BackgroundQueue(db, { concurrency: 1 }),
```

Comment:
Since this background queue isn't shared, perhaps it can be created internally to `VerificationListener`.  That would also allow `VerificationListener` to ensure `concurrency: 1` if it relies on that.